### ISSUE TITLE
Dialog: blurred text

### ISSUE THREAD

**Issue by moxival** [telerik/kendo-angular#227]
There is some weird bug with the dialog component, the text (and anything inside) gets blurred when animating something (.fa-spin or any css animation) or simply by having a kendo switch inside with odd dialog height (like 123px)

Here is a plunker:
http://plnkr.co/edit/gb6bQWbY7vvnHN8Hrc8w

It has probably something to do with the 
```css
-webkit-transform: translate(-50%,-50%);
transform: translate(-50%,-50%);
```
These might help:
https://forum.webflow.com/t/a-fix-for-when-your-elements-move-up-and-or-blur-during-animations-interactions-on-chrome/11629
http://stackoverflow.com/questions/6411361/webkit-based-blurry-distorted-text-post-animation-via-translate3d
http://stackoverflow.com/questions/31109299/css-transform-translate-50-50-makes-texts-blurry

**Comment by moxival** [comment]
Hey guys is this fixed - i am still having this problem

**Comment by gyoshev** [comment]
@moxival I'm afraid this is not fixed yet, but will be in an upcoming version of the dialog. An immediate workaround is to apply the styles from the referenced commit.

**Comment by gyoshev** [comment]
The issue has been fixed in version `0.21.0` of the dialog.
